client/src/router.ts

```diff
@@ -0,0 +1,17 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import Login from './views/Login.vue'
+import SignUp from './views/SignUp.vue'
+import Dashboard from './views/Dashboard.vue'
+
+const routes = [
+  { path: '', name: 'Login', component: Login },
```

Comment:
path: '/login', and if the user is logged in add a redirect to the dashboard, otherwise to the login